collectors/PyVimServiceCollector.py

```diff
@@ -0,0 +1,34 @@
+import logging
+from os import getenv
+
+from prometheus_client.core import GaugeMetricFamily
+
+from BaseCollector import BaseCollector
+
+# init logging
+logger = logging.getLogger('esxi-exporter')
+
+
+class PyVimServiceCollector(BaseCollector):
+
+    def __init__(self) -> None:
+        super().__init__()
+
+    def collect(self):
+        """
+        Collects information about critical esxi host services
+        To be registered at a prometheus http server
+        :return: yield GaugeMetricFamily
+        """
+
+        gauge_metric = GaugeMetricFamily('esxi_pyvim_service_state', '1=running, 0=stopped',
+                                         labels=['vcenter', 'host', 'service'])

```

Comment:
It just came to my mind to rename the label `host` to `hostsystem`. In that way it would be more comparable to the vrops-exporter and elastic-query-exporter.
Please adjust also in the other collectors.

Reply:
done.